Python/sort/Bucket_Sort.py

```diff
@@ -0,0 +1,61 @@
+# Bucket Sort Algorithm
+
+# Function to perform Insertion Sort
+def InsertionSort(array, size):
+        
+    for index1 in range(1, size):
+        
+        value = array[index1]
+        index2 = index1 - 1
+            
+        while index2 >=0 and array[index2] > value : 
+            
+            array[index2+1] = array[index2] 
+            index2 -= 1
+            
+        array[index2+1] = value
+      
+    return array   
+
+# Function to perform Bucket Sorting
+def BucketSort(unsortedArray, size):
+    
+    Bucket = []  

```

Comment:
Follow single convention for variable naming. Will suggest to follow PEP8 Guidelines
https://www.python.org/dev/peps/pep-0008/